adapters/flipp/flipp.go

```diff
@@ -25,23 +26,26 @@ const (
 	defaultCurrency = "USD"
 )
 
-var uuidGenerator uuidutil.UUIDGenerator
-
 var (
 	count    int64 = 1
 	adTypes        = []int64{4309, 641}
 	dtxTypes       = []int64{5061}
 )
 
 type adapter struct {
-	endpoint string
+	endpoint      string
+	uuidGenerator uuidutil.UUIDGenerator
 }
 
+var (
+	errRequestEmpty = errors.New("adapterRequest is empty")
+)
+
 // Builder builds a new instance of the Flipp adapter for the given bidder with the given config.
 func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
-	uuidGenerator = uuidutil.UUIDRandomGenerator{}
```

Comment:
The `uuidGenerator` should be scoped to the adapter instance per our rules. This is to avoid memory access issues if there are adapter aliases specified.